example_komunikaty.py

```
@@ -0,0 +1,56 @@
+
+    def stop(self):
+        if self.is_moving:
+            self.log("Robot stopped", "info")
+            self.is_moving = False
+        else:
+            self.log("Stop command received, but robot was not moving", "info")
+
+#-----------------------------------------------------------------------------
+#TO MACIE W DUPIE, ROBICIE TYLKO TO CO WYŻEJ
+#jak to ma wygladac w pliku strona.py:
+@app.route('/start', methods=['POST'])
```

Ruff (0.11.9)

Undefined name `app`

(F821)